hs_tracking/management/commands/stats.py

```diff
@@ -197,12 +197,12 @@ def yesterdays_variables(self, lookback=1):
             vals = self.dict_spc_to_pipe(v.value)
 
             # encode variables as key value pairs (except for timestamp)
-            values = [unicode(v.timestamp).encode('utf-8'),
-                      'user_id=%s' % unicode(uid).encode(),
-                      'session_id=%s' % unicode(v.session.id).encode(),
-                      'action=%s' % unicode(v.name).encode(),
+            values = [str(v.timestamp).encode('utf-8'),
```

Comment:
default is utf-8. Why do we specify it half the time, and the other half, default it?

Reply:
There were about 10 that specified utf-8, I'll remove utf-8 when calling encode across the project